docs/backup-restore.md

```diff
@@ -0,0 +1,109 @@
+
+## Full backup
+
+Full backup using `mydumper` to take the logical backup of TiDB cluster. The backup job will create a PVC to store backup data.
+
+By default, the backup uses PV to store the backup data. You can also store the backup data to [Google Cloud Storage](https://cloud.google.com/storage/) bucket or [Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/) by changing the configuration. This way the PV temporarily stores backup data before it is placed in object storage. Refer to [TiDB cluster Backup configuration](./references/tidb-backup-configuration.md) for full configuration guide of backup and restore.
+
+You can either setup a scheduled full backup or take a full backup in ad-hoc manner.
+
+### Scheduled full backup
+
+Scheduled full backup is created along side the TiDB cluster, and it runs periodically like the crontab job.
+
+To configure a scheduled full backup, modify the `scheduledBackup` section in the `charts/tidb-cluster/values.yaml` of the tidb cluster:
+
+* Set `scheduledBackup.create` to `true`
+* Set `scheduledBackup.storageClassName` to the PV storage class name used for backup data
+
+> **Note:** You must set the scheduled full backup PV's [reclaim policy](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy) to `Retain` to keep your backup data safe.
+
+* Configure `scheduledBackup.schedule` in the [Cron](https://en.wikipedia.org/wiki/Cron) format to define the scheduling
+* `scheduledBakcup.user` and `scheduledBackup.password` must be set to the correct user which has the permission to read the database to be backuped.
+
+Then, create a new cluster with the scheduled full backup configured by `helm install`, or enabling scheduled full backup for existing cluster by `helm upgrade`:
+
+```shell
+$ helm upgrade ${RELEASE_NAME} charts/tidb-cluster -f charts/tidb-cluster/values.yaml
+```
+
+### Ad-Hoc full backup
+
+Ad-hoc backup runs to complete for once. This functionality is encapsulated in another helm chart, `charts/tidb-backup`. According to the `mode` in `charts/tidb-backup/values.yaml`, this chart can perform either full backup or restore. We will cover restore operation in the [restore section](#restore) of this document. 
+
+To create an ad-hoc full backup job, modify the `charts/tidb-backup/values.yaml`:
+
+* Set the `clusterName` to the target TiDB cluster name
+* Set `mode` to `backup`
+* Set `storage.className` to the PV storage class name used for backup data
+* Adjust the `storage.size` according to your database size
+
+> **Note:** You must set the ad-hoc full backup PV's [reclaim policy](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy) to `Retain` to keep your backup data safe.
+
+Create a secret containing the user and password that has the permission to backup the database:
```

Comment:
```suggestion
Create a secret containing the user and password that has the privilege to back up the database:
```

Reply:
`a secret` what? Readers will feel that you forget one word. Cloud you explain what `secret` means here?

Reply:
It's a noun in Kubernetes context, actually I mean "Create a Kubernetes [Secret]() containing...". 
I will make it clear.